src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs

```diff
@@ -43,7 +43,10 @@ internal RenderEffect(global::System.IntPtr cPtr) : base(cPtr)
         /// <remarks>
         /// Created RenderEffect is immutable.
         /// </remarks>
-        /// <param name="blurRadius">The blur radius value. The unit is pixel for standard cases.</param>
+        /// <param name="blurRadius">
+        /// The blur radius value. The unit is pixel for standard cases.
+        /// If the value is too big, blur may fail due to GPU constraints.
```

Comment:
Just my curiosity :
How big shouldn't the value be? 😂

Reply:
I checked GL_MAX_VERTEX_UNIFORM_VECTORS for ubuntu desktop and pontusm. It was like 1024 or 4096(quite old memory).. Details are at [Uniform's documentation](https://www.khronos.org/opengl/wiki/Uniform_(GLSL)#Implementation_limits).
I just tried to invoke the same error but failed. T.T Sorry for this ambiguous explanation....

Reply:
maybe I can remove this edits. Thanks.

Reply:
Oh!!! I just wanted to ask the amount ...!
Later, you can add the document in the description or the comment like this :)
Thank you for answering my question!!